### ISSUE TITLE
A16 random panic: AMCC PLANE1 Unexpected RT traffic hitting DT or DT lock

### ISSUE THREAD

**Issue by leg1tsoul** [opa334/Dopamine#388]
I'm getting random kernel panic's while jailbroken.
Mostly they happen when i play Call Of Duty Mobile.
Any solution? The crash logs seems like it's not spinlock panic everyone was talking about

Crash logs are attached.
[logs.zip](https://github.com/opa334/Dopamine/files/14334940/logs.zip)

**Comment by leg1tsoul** [comment]
Also, another panic you may be interested in: 
[panic-full-2024-02-19-193954.000.zip](https://github.com/opa334/Dopamine/files/14335769/panic-full-2024-02-19-193954.000.zip)
I assume it happens after i try to re-jailbreak after a kernel panic, because jailbreak doesnt happen after kernel panic (it reboots, but no Sileo app and i assume kernel panic happens at that time. However, if i jailbreak again everything works)

**Comment by Don021123** [comment]
Is it solved? I have the same problem.

**Comment by opa334** [comment]
It is not solved, since I lack an A16 device on a compatible firmware.

**Comment by Don021123** [comment]
I truly appreciate your answer. Having used the A16 device for over a year, this particular issue has manifested itself merely on a single occasion. From what I've gathered through online research, it appears plausible that moisture infiltration might have affected the light sensor—a theory that aligns with my habit of using the device during showers.

**Comment by opa334** [comment]
So the issue is something is triggering a prefetch abort on some coprocessor memory. If I knew what / where this memory was I could avoid mapping it in, but I don't. And also still don't really have a test device.

**Comment by EliasTekdin** [comment]
Hello @opa334 , I recently updated Dopamine to 3.0.4 today and was using my phone normally, when another sudden and unexpected reboot occurred on my device.

[panic-full-2026-08-10-132340.0002.ips.txt](https://github.com/user-attachments/files/30897468/panic-full-2026-08-10-132340.0002.ips.txt)

<img width="1290" height="2796" alt="Image" src="https://github.com/user-attachments/assets/3ea8b6ec-87c2-4ee5-b3e5-4034bd44ec15" />

**Comment by opa334** [comment]
> Hello @opa334 , I recently updated Dopamine to 3.0.4 today and was using my phone normally, when another sudden and unexpected reboot occurred on my device.
> 
> [panic-full-2026-08-10-132340.0002.ips.txt](https://github.com/user-attachments/files/30897468/panic-full-2026-08-10-132340.0002.ips.txt)
> 
> <img width="1290" height="2796" alt="Image" src="https://github.com/user-attachments/assets/3ea8b6ec-87c2-4ee5-b3e5-4034bd44ec15" />
> 

Different issue, I suggest making a separate issue for tracking that

**Comment by Don021123** [comment]
Thanks so much for looking into this. With 3.0.7 out, does that mean the panic is completely fixed now?

**Comment by opa334** [comment]
> Thanks so much for looking into this. With 3.0.7 out, does that mean the panic is completely fixed now?

The data.kalloc.96 and potentially the sfree panic should be fixed in 3.0.7. The panic that this issue was originally about is not fixed and likely won't be since I lack a test device.

**Comment by Lessica** [comment]
It is currently known that the frequency of this panic is significantly amplified in Full physrw access mode. I have several physical devices that fall within the A17 target range. If you need debugging assistance or help with testing, please be sure to reach out to me on Discord—I’d be happy to help (if I can).

**Comment by Lessica** [comment]
> So the issue is something is triggering a prefetch abort on some coprocessor memory. If I knew what / where this memory was I could avoid mapping it in, but I don't. And also still don't really have a test device.

Yes, my real-device experiments also support this. Even when only accessing pages adjacent to RT, AMCC is triggered.
